src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderNoPackage.java

```diff
@@ -1,9 +1,30 @@
-//non-compiled with javac: contains specially crafted set of imports for testing
+//non-compiled with javac: contains no package for testing
 import java.io.*;
 
 
-import java.util.*;
+import java.util.*; //warn
+
+import java.util.HashMap; //warn
+import java.util.LinkedList;
+// comments between imports
+
+import javax.net.ServerSocketFactory; //warn
+
+// comments between imports
+import javax.net.SocketFactory; //no warns until https://github.com/checkstyle/checkstyle/issues/7314
```

Comment:
as I suggested at https://github.com/checkstyle/checkstyle/pull/7366#pullrequestreview-334235723 and to avoid such `until` please put in this PR commit from referenced PR. Referenced PR is on latest phase of review, so not that much changed are expected in it. Ones PR is merged you can remove commit and rebase.

I really want this PR be the last for extra spaces.

Reply:
Sorry for misreading your comment :( 
Fixed.

Reply:
Ok, lets wait for @rnveach  final review in first PR... looks like we are close to merge.